recipes/ldraw/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set parts_version = "20190911" %}
+{% set mklist_version = "1.6" %}
+{% set build_number = "0" %}
+
+package:
+  name: ldraw
+
+source:
+  - url: http://www.ldraw.org/library/updates/complete.zip
```

Comment:
Is there a stable link?

Reply:
But I can tell that other packages like apt-get, and freebsd ports also get the source from that URL.

Reply:
What happens when there's an update? The file will be updated and we won't be able to download the old version and the new version's SHA256 will be different

Reply:
When there is an update

 - a new `lcadXXXX.zip` is created with *only the new parts*.
 - the file `complete.zip` is replaced with the new version including the new parts.

Reply:
Wow. They sure know ["How to make package managers cry"](https://www.youtube.com/watch?v=NSemlYagjIU)

Reply:
I suggest hosting the source tarball somewhere with the version in the name and using it.

Reply:
> Wow. They sure know "How to make package managers cry"

Yes, I can make a repository to host the tarballs.... Will update the recipe this week.

Reply:
I should also post on their forum to report our issues.

Reply:
> I should also post on their forum to report our issues.

Reported upstream: https://forums.ldraw.org/thread-23671-post-34002.html#pid34002